packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -760,41 +756,6 @@ describe('webex-core', () => {
             assert.isRejected(waitForService);
           });
         });
-
-        describe('when the service will exist', () => {
```

Comment:
These are invalid tests. There is no guarantee that the name of the hosts or the priority of a host won't change from different calls to u2c. Right now, wdm is changing the priority of the hosts in the EU cluster. (I assume for load balancing reasons).